### PR TITLE
Added tests for zone endpoint

### PR DESCRIPTION
**Pull request by sam-i-am012**
Addresses #26

### CODE REVIEW

File: src/test/java/org/example/Operations/ZoneOperationsTest.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                }
+            }
+
+            // Remove the remote zone.
+            res = rods.zoneOperations().remove(rodsToken, zoneName);
+            // logger.debug(res.getBody());
+            assertEquals("Removing zone request failed", 200, res.getHttpStatusCode());
+            assertEquals("Removing zone failed", 0,
+                    getIrodsResponseStatusCode(res.getBody()));
+
```
Comment (korydraughn) [inline src/test/java/org/example/Operations/ZoneOperationsTest.java:88]: Remove empty line.

**Review by korydraughn — COMMENTED** [review]
Looks good. Just one thing to handle.

Go ahead and squash once the review comment is resolved.

File: src/test/java/org/example/Operations/ZoneOperationsTest.java (hunk 1 above)
Comment (sam-i-am012) [reply src/test/java/org/example/Operations/ZoneOperationsTest.java:88]: Done.

**Comment by sam-i-am012** [comment]
Done.

**Review by korydraughn — APPROVED** [review]
If the tests are passing, pound it.

**Comment by korydraughn** [comment]
Merging.

**Pull request merged**